__tests__/app.test.js

```diff
@@ -384,6 +386,39 @@ describe('GET/api/reviews/review_id', () => {
     });
 });
 
+describe('DELETE/api/comments/:comment_id', () => {
+    test('returns a 204', () => {
+        return request(app).delete('/api/comments/1')
+        .expect(204)
+    });
+    test('deletes the comment specified in params', () => {
+        return request(app).delete('/api/comments/1')
+        .expect(204)
+        .then((body) => {
+            expect(body.noContent).toBe(true)
+            return request(app).get('/api/reviews/2/comments')
+        })
+        .then(({body}) => {
```

Comment:
I'd iterate through the comments and check that their ID doesn't match the one which you deleted.